tsl/src/continuous_aggs/invalidation.c

```diff
@@ -147,14 +199,16 @@ static InvalidationResult
 cut_invalidation_along_refresh_window(const CaggInvalidationState *state, int64 modification_time,
 									  int64 lowest_modified_value, int64 greatest_modified_value,
 									  const InternalTimeRange *refresh_window,
-									  const ItemPointer tid)
+									  const ItemPointer tid, Invalidation *remainder)
```

Comment:
Nit: Usually good to be consistent on where the "object" being manipulated is placed. You are placing the object last here, but first in the other functions.

Reply:
There is no object being modified. If you mean `remainder`, it is a return value/result, i.e., the remainder after the cut.

Reply:
Yes, it says to cut the invalidation along the refresh window, which suggests that the invalidation is the object. Either way it is a nit.

Reply:
So, the remainder is not the invalidation that gets cut in this case. The remainder is the return value for the part of the invalidation that fits within the window. I tried to clarify this in the code.